src/modules/auth/decorators/get-user.decorator.ts

```diff
@@ -5,11 +5,13 @@ import {
 } from '@nestjs/common';
 
 export const GetUser = createParamDecorator(
-  (data: unknown, ctx: ExecutionContext) => {
+  (data: string | undefined, ctx: ExecutionContext) => {
     const request = ctx.switchToHttp().getRequest();
+
     if (!request.user) {
-      throw new UnauthorizedException('User not found');
+      throw new UnauthorizedException('User is not authenticated');
```

Comment:
Please create error messages to use globally. There are the same type of exceptions that are using different messages right now. 
![image](https://github.com/user-attachments/assets/75df48cf-b80d-48e1-a46e-036d8165a877)